pkg/controller/jobs/deployment/deployment_webhook.go

```diff
@@ -57,16 +57,16 @@ func SetupWebhook(mgr ctrl.Manager, opts ...jobframework.Option) error {
 var _ admission.CustomDefaulter = &Webhook{}
 
 func (wh *Webhook) Default(ctx context.Context, obj runtime.Object) error {
-	d := fromObject(obj)
-	log := ctrl.LoggerFrom(ctx).WithName("deployment-webhook").WithValues("deployment", klog.KObj(d))
+	deployment := fromObject(obj)
+
+	log := ctrl.LoggerFrom(ctx).WithName("deployment-webhook").WithValues("deployment", klog.KObj(deployment))
```

Comment:
```suggestion
	log := ctrl.LoggerFrom(ctx).WithName("deployment-webhook")
```
controller-runtime is adding this to the logger before this call.

Reply:
Thanks! I will remove it in separate PR.